fastfood/food.py

```diff
@@ -141,85 +151,29 @@ def update_cookbook(cookbook, templatepack, stencilset_name, **options):
     return written_files, cookbook
 
 
-def create_new_cookbook(cookbook_name, templatepack,
-                        cookbooks_home, force=False):
+def create_new_cookbook(cookbook_name, cookbooks_home, force=False):
```

Comment:
I think I see what you mean. Let me fix this so force is actually used correctly in `process_stencil`.